Source/WebCore/platform/text/TextSpacing.cpp

```diff
@@ -103,6 +104,12 @@ static bool isNonIdeographicNumeral(char32_t character, uint32_t generalCategory
 // Classes are defined at https://www.w3.org/TR/css-text-4/#text-spacing-classes
 CharacterClass characterClass(char32_t character)
 {
+    // fast path
+    if (isASCIILower(character) || isASCIIUpper(character))
```

Comment:
Is this check distinct from using a single `isASCIIAlpha(character)`?

Reply:
Oops, I haven't seen that function. It is exactly the same thing. Thanks.